cursorless-talon/src/cursorless.talon

```diff
@@ -24,3 +24,8 @@ cursorless instructions:   user.cursorless_open_instructions()
 cursorless docks:          user.cursorless_open_instructions()
 cursorless reference:      user.cursorless_cheat_sheet_show_html()
 cursorless cheat sheet:    user.cursorless_cheat_sheet_show_html()
+cursorless settings:
+    sleep(50ms)
+    user.vscode("workbench.action.openGlobalSettings")
+    sleep(250ms)
```

Comment:
Not in love with this and the other arbitrary sleep 🤔

Reply:
We already have those for other commands so I guess it's fine until we have something better, but why the leading sleep?

Reply:
Yeah good question.  I have it in my personal "show settings" command.  Presuming I added it for a reason 🤷

Reply:
Feels like a left over from something else maybe? I would remove it.